client/components/PubEdge/PubEdge.js

```diff
@@ -0,0 +1,63 @@
+import dateFormat from 'dateformat';
+import React, { useCallback, useState } from 'react';
+import { Button } from 'reakit/Button';
+
+import { Byline } from 'components';
+import { pubEdgeType } from './constants';
+
+require('./pubEdge.scss');
+
+export const propTypes = {
+	pubEdge: pubEdgeType.isRequired,
+};
+
+const PubEdge = (props) => {
+	const {
+		pubEdge: { avatar, contributors, description, publicationDate, title, url },
+	} = props;
+	const [open, setOpen] = useState(false);
+	const handleToggleDescriptionClick = useCallback(
+		(e) => {
+			e.preventDefault();
+			setOpen(!open);
+		},
+		[open],
+	);
+	const publishedAt = dateFormat(publicationDate, 'mmm dd, yyyy');
```

Comment:
After writing this exact line out about 100 times I finally had enough and wrote `formatDate` from `utils/dates.js`, which uses this date formatting as its default. I'd encourage you to do the same!